### PR TITLE
Feature/55655 mattress type app is

### PR DESCRIPTION
**Pull request by Sanid1707**
This PR implements all APIs for Mattress Type :
The Functionality includes the following :

- CRUD functionality 
- Displaying the Type Mattress is split into 2 APIS to avoid over fetching and prevent backend load

### CODE REVIEW

File: Backend/Backend/DTOs/MattressType/MattressTypeTileDTO.cs
Comment (Terrie-png) [inline Backend/Backend/DTOs/MattressType/MattressTypeTileDTO.cs:1]: why create a duplication of dto ?

File: Backend/Backend/appsettings.json
Comment (Terrie-png) [inline Backend/Backend/appsettings.json:1]: Please dont change the appsetting to the main branch i need this for deployment on docker

**Review by Terrie-png — CHANGES_REQUESTED** [review]
Can revert the changes from appsetting ?

File: Backend/Backend/DTOs/MattressType/MattressTypeTileDTO.cs
Comment (Sanid1707) [reply Backend/Backend/DTOs/MattressType/MattressTypeTileDTO.cs:1]: To. make the backend scalable and prevent over fetching I have separated the displaying the Mattress Details into 2 APIS and hence there are 2 DTOs to make this process more readable . Each DTO serves a specific purpose, tailored for the data needed by a specific API.

**Review by Terrie-png — DISMISSED** [review]
Everything looks fine

**Review by Terrie-png — DISMISSED** [review]
Looks good now

**Review by Terrie-png — APPROVED** [review]
ok

**Pull request merged**